specs/SingleAppHost.md

```diff
@@ -0,0 +1,61 @@
+# Background
+While WebViews from different app processes normally don't share the same WebView browser process instance, app processes from the same product suite may share the same webview browser process
+instance by specifying the same user data folder when creating WebView2Environment object.
+The `AreMultipleHostProcessesAllowed` property is for the developer to express the sharing intent, so that we could provide optimized security and performance according expected usage.
+
```

Comment:
This feature is about whether the browser process can be shared or not, but the name sounds like it's about the host process. I'd find this much easier to understand if it were named `AllowBrowserProcessSharing`, or to make it default to false name it `PreventBrowserProcessSharing`.

But reading further, the whole thing about browser process sharing seems to be an implementation detail. Really this feature is about whether the host is getting exclusive access to the user data folder. So suggest `ExclusiveUserDataFolderAccess`.

Reply:
I'll let review board decide what name is easier for developers to understand. So, we would change to the last suggested name in this review(which is ExclusiveUserDataFolderAccess for now).

I personally feel that ExclusiveUserDataFolderAccess is too technical and like PreventBrowserProcessSharing better.

Reply:
I think you need to understand the implementation details to understand what PreventBrowserProcessSharing really means. The key for the developer to understand is that they're opening the user folder for exclusive use or not. Files have the concept of being opened in exclusive mode or shared mode, which is basically the concept here.

Reply:
OK, will go with ExclusiveUserDataFolderAccess

Reply:
Sounds good!

Reply:
Changed the name to ExclusiveUserDataFolderAccess